sympy/concrete/summations.py

```diff
@@ -1135,7 +1135,7 @@ def _eval_sum_hyper(f, i, a):
     bq = params[1]
     x = ab[0]/ab[1]
     h = hyper(ap, bq, x)
-
+    f = combsimp(f)
```

Comment:
How about putting this in `eval_sum_hyper`? It may call `_eval_sum_hyper` twice, and `combsimp` will be repeated. If it were applied in `eval_sum_hyper`, a single call would suffice.

Reply:
I have tried it but if we put it in `eval_sum_hyper` then some of the tests will fail eg: `Sum(a**-i/(a - b), (i, 0, n)).doit() `or `Sum((b*a**i - c*a**i)**-2, (i, 0, n))  `

Reply:
These failures are not necessarily serious. If only the form of the result has changed, it is possible to edit the test. But it seems that there are other issues as well. Hence it may be best to keep `combsimp` here for now.